app/pb_kits/playbook/packs/examples.js

```diff
@@ -78,12 +78,6 @@ import * as Toggle from 'pb_toggle/docs'
 import * as User from 'pb_user/docs'
 import * as UserBadge from 'pb_user_badge/docs'
 
-import PbTypeahead from 'pb_typeahead'
-PbTypeahead.start()
-
-import PbPopover from 'pb_popover'
-PbPopover.start()
```

Comment:
These should never be here since we are importing them via `application.js` which imports `vendor.js` in Playbook and is used by Nitro, Huddle and Playgrounds.